index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="styles.css">
+
+    <nav id="navbar" class="nav">
+        <ul class="nav-list">
+          <li>
+            <a href="#welcome-section">About</a>
+          </li>
+          <li>
+            <a href="#projects">Work</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+</head>
+<body>
+    <section id="welcome-section" class="welcome-section">
+        <h1>Hi we are Valentina and Nestor</h1>  
+        <br> Thats our projects</br>  
+    </section>
+
+    <section id="projects" class="projects-section">
+        <h2 class=""projects-section-header">Projects:</h2>
+        <p class="project-title">
+            Project 1
+        </p>
+    </section>
+
+    <section id="contact" class="contact-section">
+        <a
+          href="https://facebook.com/"target="_blank">Facebook
+	    </a>
```

Comment:
Your portfolio should have an **_a_** element with an id of profile-link

Reply:
- Just as I suggested earlier, it will be better to make your suggestions sound more polite than as an order. Can we edit your comment to meet this requirement? 

- Also, kindly state the reasons WHY the change needs to be implemented 🌸